style.css

```diff
@@ -1,3 +1,88 @@
 *{
     box-sizing: border-box;
 }
+body{
+    background-color: #ccc;
+  background-color: #ccc;
+  font-family: Arial, Helvetica, sans-serif;
+  min-height: 100vh;
+ }
+ .contanier{
+     width:500px;
+     max-width: 100%;
+     margin: 0 auto;
+ }
+ header{
+     border:3px solid #000;
+     display: flex;
+     border-radius: 20px;
+     justify-content: space-between;
+     padding: 25px;
+     margin-bottom: 50px;
+ }
+ h1{
```

Comment:
for the next time >> you should always use classes don't use the tags for the style

Reply:
ok